src/main/java/eu/merloteducation/organisationsorchestrator/service/OrganizationMetadataService.java

```diff
@@ -48,9 +48,16 @@ public MerlotParticipantMetaDto updateMerlotParticipantMeta(MerlotParticipantMet
         return mapper.organizationMetadataToMerlotParticipantMetaDto(repository.save(dbMetadata));
     }
 
-    public List<MerlotParticipantMetaDto> getParticipantsByMembershipClass(MembershipClass membershipClass){
+    public List<MerlotParticipantMetaDto> getParticipantsByMembershipClass(MembershipClass membershipClass) {
+
         List<OrganizationMetadata> orgaMetadataList = repository.findByMembershipClass(membershipClass);
-        return orgaMetadataList.stream().map(orgaMetadata -> mapper.organizationMetadataToMerlotParticipantMetaDto(orgaMetadata)).toList();
+        return orgaMetadataList.stream()
+            .map(orgaMetadata -> mapper.organizationMetadataToMerlotParticipantMetaDto(orgaMetadata)).toList();
+    }
+
+    public List<String> getInactiveParticipants() {
```

Comment:
This probably should be renamed to something like getInactiveParticipantIds since we are not returning whole metadata objects